src/api/spec/controllers/webui/architectures_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ArchitecturesController do
+  let(:confirmed_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+
+  describe 'GET #index' do
```

Comment:
I don't like this. We should test that this controller has the require_admin before_filter and that Webui::WebuiController#require_admin works